### PR TITLE
Add tree obstacle

### PR DESCRIPTION
**Pull request by Tobeast1**

### CODE REVIEW

**Review by lawin666 — CHANGES_REQUESTED** [review]
I think the collisionshape is a little bit too big. U can make it to a rectangle instead n make it as big as the tree trunk.

**Comment by daogiabaohp420** [comment]
I think you can put your collisionshape2D down only around the trunk of tree so it make the game more 2D when player stand behind the tree.

File: README.md
Comment (xix-xeaon) [inline README.md:1]: This file shouldn't be removed.

File: icon.svg
Comment (xix-xeaon) [inline icon.svg:1]: This file shouldn't be removed either.

File: Trees/foliagePack_005.png.import
Comment (xix-xeaon) [inline Trees/foliagePack_005.png.import:1]: This change doesn't make sense, I think this commit is incorrect.

File: Trees/pine_tree.tscn [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[gd_scene load_steps=4 format=3 uid="uid://cvxioub202qqm"]
+
+[ext_resource type="Texture2D" uid="uid://b7yj4xkkha1xm" path="res://Trees/ground_shadow.png" id="1_3ncjd"]
+[ext_resource type="Texture2D" uid="uid://6olnvg3y2rq" path="res://Trees/foliagePack_005.png" id="1_lqnct"]
+
+[sub_resource type="RectangleShape2D" id="RectangleShape2D_lyxiq"]
+size = Vector2(26, 61)
+
+[node name="StaticBody2D" type="StaticBody2D"]
```
Comment (xix-xeaon) [inline Trees/pine_tree.tscn:9]: The name "StaticBody2D" is going to make it confusing to use later on.

**Review by xix-xeaon — CHANGES_REQUESTED** [review]
This PR needs a bit of cleanup but otherwise I think it's fine and could be merged.

**Review by xix-xeaon — APPROVED** [review]
LGTM

**Pull request merged**